scspell/_util.py

```diff
@@ -60,8 +60,20 @@ def open_with_encoding(filename, encoding=None, mode='r'):
 
 def detect_encoding(filename):
     """Return file encoding."""
+
+    # Thanks again stackoverflow for python 2 v 3,
+    # http://stackoverflow.com/questions/15032108/pythons-open-throws-different-errors-for-file-not-found-how-to-handle-b
+    fnotfound = getattr(__builtins__, 'FileNotFoundError', IOError)
```

Comment:
I think simply using `except (IOError, OSError):` would be much more elegant and easier to understand.